faker/providers/ssn/fi_FI/__init__.py

```diff
@@ -7,7 +7,7 @@
 
 class Provider(SsnProvider):
 
-    def ssn(self, min_age=18, max_age=90):
+    def ssn(self, min_age=0, max_age=105):
         """
         Returns 11 character Finnish personal identity code (Henkilötunnus,
         HETU, Swedish: Personbeteckning). Age of person is between 18 and 90
```

Comment:
this docstring will need to be updated to say `Age of person is between 18 and 90 years by default`

Reply:
:+1: good catch. Cheers. Fix incoming